ddtrace/contrib/redis/__init__.py

```diff
@@ -1,29 +1,44 @@
-"""Instrument redis to report Redis queries.
+"""
+Traces redis client queries.
+
+If you are not autoinstrumenting with ``ddtrace-run`` then install the redis
+instrumentation with::
+
+    from ddtrace import patch
+    patch(redis=True)
```

Comment:
`patch_all`?

or does this follow other integrations?

Reply:
🤷‍♂️ I don't like having it mentioned at all in each integration. We should really just point to a section of the docs for installing/patching integrations